.github/actions/prepare-minimega/action.yml

```diff
@@ -0,0 +1,29 @@
+# A composite action to install and initialize minimega
+
+name: Prepare minimega
+
+description: Install and initialize minimega
+
+runs:
+  using: composite
+  steps:
+    - name: Install minimega
+      run: |
+        wget https://github.com/sandia-minimega/minimega/releases/download/2.9/minimega-2.9.deb
+        sudo dpkg -i minimega-2.9.deb
+        sudo chown -R $USER:minimega $MM_INSTALL_DIR
+        sudo ln -s $MM_INSTALL_DIR/bin/minimega /usr/local/bin/minimega
+        sudo ln -s $MM_INSTALL_DIR/bin/minimega /usr/local/bin/mm
```

Comment:
As a note: I did not include dependency installation for either minimega, discovery, or FIREWHEEL in the preparation scripts. I found that each call to sudo apt update and sudo apt install had enough overhead to increase the test runtime by a decent amount. However, now that the tests are working and I'm not debugging, we may want to consider if that overhead is worth having these composite actions be self-sufficient.